File: Modul-4/C/PRAK403_2410817210001_MuhammadFadillah.c
#include <stdio.h>

int main() {
    int a, b;

    printf("Masukkan dua angka : ");
    scanf("%d %d", &a, &b);

    if (a < b) {
        for (int i = a; i <= b; i++) {
            printf("%d %d", i, b - (i - a));
            if (i != b) {
                printf(" - ");
            }
        }
    } else if (a > b) {
        for (int i = a; i >= b; i--) {
            printf("%d %d", i, b + (a - i));
            if (i != b) {
                printf(" - ");
            }
        }
    }

    printf("\n");
    return 0;
}

File: Modul-4/Python/PRAK403_2410817210001_MuhammadFadillah.py
a, b = map(int, input("Masukkan dua angka : ").split())

if a < b:
    for i in range(a, b + 1):
        print(f"{i} {b - (i - a)}", end=" - ")
elif a > b:
    for i in range(a, b - 1, -1):
        print(f"{i} {b + (a - i)}", end=" - ")

print("\b\b ")
